kedro/framework/context/context.py

```diff
@@ -1,11 +1,13 @@
+# pylint: disable=no-member
```

Comment:
Do we need to disable this for the whole file or is it possible to add it to the specific line(s) it applies to?

Reply:
Yes, I change it back to the specific line(s). It's not clear why pylint complains about "no-member" for `config_loader` only but not the other attributes.